### PR TITLE
[ZEPPELIN-3580] Adds rename note method to rest api

### PR DESCRIPTION
**Pull request by oxygen311**
### What is this PR for?
Adds rename note method to rest api

### What type of PR is it?
Improvement

### What is the Jira issue?
issue on Jira: https://issues.apache.org/jira/browse/ZEPPELIN-3580

### Questions:
- Does the licenses files need update? No
- Is there breaking changes for older versions? No
- Does this needs documentation? No

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
LGTM

File: zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java [hunk 1]
```diff
@@ -14,13 +14,20 @@
...
  */
+
 package org.apache.zeppelin.rest;
 
 import com.google.common.collect.Sets;
 import com.google.common.reflect.TypeToken;
 import com.google.gson.Gson;
 
 import org.apache.commons.lang3.StringUtils;
+import org.apache.zeppelin.rest.message.CronRequest;
```
Comment (jongyoul) [inline zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java]: Can you check the order of dependencies? It seems to be wrong

Comment (Leemoonsoo) [reply zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java]: btw, @jongyoul how about update [_tools/checkstyle.xml](https://github.com/apache/zeppelin/blob/master/_tools/checkstyle.xml) to automatically check import order?

Comment (jongyoul) [reply zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java]: I think it would be better, actually.

Comment (Leemoonsoo) [reply zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java]: Personally, I'm okay with any format / criteria that checkstyle plugin can forces.

**Comment by jongyoul** [comment]
LGTM

**Comment by jongyoul** [comment]
@oxygen311 Could you please resolve conflicts? I recommend you not to change the unrelated code to avoid merge conflicts. After resolving, I'll merge it

**Comment by oxygen311** [comment]
@jongyoul Done

**Pull request closed without merging**